src/components/countrycard/index.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;1,100;1,700&display=swap');
+
+.card {
+  background-color: #3f62a5;
+  color: #fdfdf8;
+  font-family: 'Lato', sans-serif;
+  margin: auto;
+}
+
```

Comment:
- [ ] Please make the background color of country cards staggered like chess board to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/217867576-2b1cb1fb-8ba6-4984-9a86-bd62f03af54a.png)  | ![image](https://user-images.githubusercontent.com/68796232/217867547-c033025a-e49d-4c8f-9629-6a734f0bafdb.png) |